ilp-core-codecs/src/main/java/org/interledger/core/asn/codecs/AsnInterledgerAddressCodec.java

```diff
@@ -37,6 +37,6 @@ public InterledgerAddress decode() {
 
   @Override
   public void encode(InterledgerAddress value) {
-    setCharString(value.getValue());
+    setCharString(value.value());
```

Comment:
So, the author of the immtuables library suggest not using the `get`/`set` prefix on immtuable property values because it collides with the Java POJO standard, but immutables are expressly _not_ POJOs (I only recently came across this information, which is why you see `getX` employed everywhere else in this project).

That said, if there some usefulness in providing a `getX`, I wouldn't be opposed to it. What's the magic that it gives us?

Reply:
I'm going to merge this PR as-is. If you want me to add `getValue`, mind creating an issue for it?

Reply:
That’s a good question - any opinion? Should we use getXyz(), or just .xyz()?

Reply:
To start with, I'd say consistency trumps any convention, be it Kotlins or Immutables. "Kotlins" is really a Java convention as well, hence it feeling weird to you, and is likely what is expected by most Java programmers? Given that, and the fact you are already using `getX()`, I'd say go with that? Can always be revisited later anyway? i.e. be consistent until you make a definitive decision?

Reply:
Yeah, I agree - I'll make a new PR to add the `get` syntax.